content/doc/tutorials/using-jenkins-to-build-a-java-maven-project.adoc

```diff
@@ -184,36 +183,37 @@ running any tests).
 . In the *This job has not been run* message box, click *Run*, then quickly
   click the *OPEN* link which appears briefly at the lower-right to see Jenkins
   building your Pipeline project. If you weren't able to click the *OPEN* link,
-  click the row on the main Blue Ocean interface to access this feature.
-
-+
-*Note:* You may need to wait several minutes for this first run to complete.
-After making a clone of your local `simple-java-maven-app` Git repository
-itself, Jenkins:
-
+  click the row on the main Blue Ocean interface to access this feature. +
+  *Note:* You may need to wait several minutes for this first run to complete.
+  After making a clone of your local `simple-java-maven-app` Git repository
+  itself, Jenkins:
 .. Initially queues the project to be built on the agent.
-.. Downloads the Maven Docker image and runs it in a container on Docker. +
-   image:tutorials/java-maven-01-downloading-maven-docker-image.png[alt="Downloading
-   Maven Docker image",width=100%]
+.. Downloads the Maven Docker image and runs it in a container on Docker.
++
+[.boxshadow]
```

Comment:
Is there a reason to not make `.boxshadow` on by default for images? 
Or is there some way to set it for an entire .adoc at once?  I don't think we want to pepper all our docs with this, but we will generally want images to have this.

Reply:
Thanks for your feedback @bitwiseman !

As an author, I think it's good to have the flexibility to choose whether or not you want this feature applied to any given image on any page. I think most images would have them (i.e. screenshots with no or minimal annotation). However, there might be the need to include diagrams (which may in themselves include screenshots), where this would look odd.

For instance (and referencing some of my older work lol), [this page](https://docs.covata.com/3.5/user/index.html) has an annotated diagram (which includes a screenshot), where applying a box shadow looks really odd. Hence, there's no border on this image.

On the other hand, most of the images on [the subsequent page](https://docs.covata.com/3.5/user/signing-in.html) have box shadows applied.

In summary (and I gather from what you suggested above), I think most images would have box shadows applied to them, but I'd certainly want the ability to disable it for specific images. We could switch the logic around (e.g. have them applied by default and then apply a `[.noboxshadow]` role to images that disables this). However, I'm concerned about the impact of this on other images throughout jenkins.io (even those which aren't part of the documentation).

> Or is there some way to set it for an entire .adoc at once?

I'm not entirely sure... I suspect so with [Ascidoctor's "docinfo" feature](http://asciidoctor.org/docs/user-manual/#docinfo-file), but as I started mentioning in [this comment on WEBSITE-416](https://issues.jenkins-ci.org/browse/WEBSITE-416?focusedCommentId=317287&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-317287), I actually tested this out (i.e. [based on this trick](http://discuss.asciidoctor.org/Borders-for-images-td1957.html)) by:
1. Adding a `docinfo.html` file to my tutorial directory with the appropriate CSS syntax.
1. Applying the `[.boxshadow]` syntax above images, only to discover that this wasn't being picked up.

I'll try this out again and'll report back.

Reply:
Hi again @bitwiseman , I'm at a bit of a loss here... I've tried adding the `docinfo.html` file with the following content:
```
<style>
.imageblock.docinfoboxshadow img {
  margin: 5px 5px 5px 5px;
  -moz-box-shadow: 0 0 15px 0px rgba(0, 0, 0, 0.5);
  -webkit-box-shadow: 0 0 15px 0px rgba(0, 0, 0, 0.5);
  box-shadow: 0 0 15px 0px rgba(0, 0, 0, 0.5);
}

.paragraph.docinfoboxshadow img {
  margin: 5px 5px 5px 5px;
  -moz-box-shadow: 0 0 15px 0px rgba(0, 0, 0, 0.5);
  -webkit-box-shadow: 0 0 15px 0px rgba(0, 0, 0, 0.5);
  box-shadow: 0 0 15px 0px rgba(0, 0, 0, 0.5);
}
</style>
```
then adding `:docinfo:`, `:docinfo: shared` or `:docinfo: shared,private` to the top of my relevant `.adoc` file,
and then specifying `[.docinfoboxshadow]` above image references, and no luck.

I'm either doing something very wrong or the docs build process is somehow not processing/picking up these `docinfo.html` files. I can see that they're definitely not in the source of the generated HTML files.

Reply:
Actually @bitwiseman , I realized in retrospect now that it would've been better to demonstrate the above as a pull request to this pull request.